sim/druid/item_sets_pve.go

```diff
@@ -725,6 +719,9 @@ var ItemSetGenesisCunning = core.NewItemSet(core.ItemSet{
 				ActionID: core.ActionID{SpellID: 1213174},
 				Label:    "S03 - Item - TAQ - Druid - Feral 4P Bonus",
 				OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+					if druid.form != Cat {
```

Comment:
Changing the check for if the current spell is one of the ones affected by the set bonus to use spellcode instead of "spell == druid.<catspell>.Spell fixes these issues.
I was actually having kernel panics when running the cat sim without using spellcode, that should work on live though so idk..
Changing this when I update the PR.